code/game/objects/items/toys.dm

```diff
@@ -321,6 +322,17 @@
 	icon_state = "comicstormman"
 	item_state = "comicstormman"
 
+/obj/item/toy/comic/outlandish_tales
+	name = "outlandish tales magazine"
+	desc = "A magazine specialized in publishing Tajaran Otherworldly Literature stories. The periodical magazine features texts submitted by amateur and established writers alike. \
+	Since it is printed using cheap, recycled paper, Outlandish Tales is sold for a couple of credits on the streets of Little Adhomai. The magazine also has a section dedicated to \
```

Comment:
```suggestion
	Since it is printed using cheap, recycled paper, outlandish tales is sold for a couple of credits on the streets of Little Adhomai. The magazine also has a section dedicated to \
```
since its only capitalized here.